app/js/utils/validations.js

```diff
@@ -0,0 +1,5 @@
+export const minValue = min => value => (value && value < min ? `Must be at least ${min}` : undefined);
+
+export const maxValue = max => value => (value && value > max ? `Must be less than ${max + 1}` : undefined);
+
+export const abnormalMaxValue = max => value => (value && value > max ? `Abnormal value` : undefined);
```

Comment:
Not a big deal, but we have these in react-components if we want to just use them there?

And let's go ahead and add abnormalMinValue as well and then use it based on the minimum value.